### PR TITLE
Dynatrace doc changes

### PR DESCRIPTION
**Pull request by hruhek**

### CODE REVIEW

File: stackpacks/integrations/dynatrace.md [hunk 1]
```diff
@@ -35,36 +35,72 @@ Install the Dynatrace StackPack from the StackState UI **StackPacks** &gt; **Int
...
-      # environment: <environment>  # default 'production'
-      # relative_time : <relative_time> # default 'hour'
-      # events_bootstrap_days: 5  # default 5 days
-      # events_process_limit: 10000  # default 10k events
-      # tags:
-      #   - foo:bar
-     ```
-2. Optional: Add a **domain** and **environment** in the `conf.yaml` file to specify where imported Dynatrace topology will end up in StackState \(default domain=dynatrace and environment=production\).
-3. [Restart the StackState Agent\(s\)](../../setup/agent/about-stackstate-agent.md#deploy-and-run-stackstate-agent-v2) to apply the configuration changes.
+There are two Dynatrace checks:
```
Comment (gggina) [inline stackpacks/integrations/dynatrace.md:38]: Is it ok to configure just one check? (e.g. to get only topology and no health)?
Would be nice to explain a bit more here

File: stackpacks/integrations/dynatrace.md [hunk 2]
```diff
@@ -35,36 +35,72 @@ Install the Dynatrace StackPack from the StackState UI **StackPacks** &gt; **Int
...
+- Topology check
+- Health check
+
+#### Dynatrace Topology Check
+
+To enable the Dynatrace topology check and begin collecting data from Dynatrace, add the following configuration to StackState Agent V2:
```
Comment (gggina) [inline stackpacks/integrations/dynatrace.md:44]: "To enable the Dynatrace topology check and begin collecting **topology** data from Dynatrace"...

File: stackpacks/integrations/dynatrace.md [hunk 3]
```diff
@@ -35,36 +35,72 @@ Install the Dynatrace StackPack from the StackState UI **StackPacks** &gt; **Int
...
+        # domain: <domain>  # default 'dynatrace'
+        # environment: <environment>  # default 'production'
+        # relative_time : <relative_time> # default 'hour'
+        # custom_device_relative_time: 1h
+        # custom_device_fields: +fromRelationships,+toRelationships,+tags,+managementZones,+properties.dnsNames,+properties.ipAddress
+        # tags:
+        #   - foo:bar
+    
+    ```
+2. Optional: Add a **domain** and **environment** in the `conf.yaml` file to specify where imported Dynatrace topology will end up in StackState (default domain=dynatrace and environment=production).
```
Comment (gggina) [inline stackpacks/integrations/dynatrace.md:71]: 2. You can also add optional configuration to specify where imported Dynatrace topology will end up in StackState:
  - **domain** - The domain to use for imported topology (default `dynatrace`).
  - **environment** - The environment to use for imported topology  (default `production`).

File: stackpacks/integrations/dynatrace.md [hunk 4]
```diff
@@ -35,36 +35,72 @@ Install the Dynatrace StackPack from the StackState UI **StackPacks** &gt; **Int
...
+        # cert: /path/to/cert.pem
+        # keyfile: /path/to/key.pem
+        # timeout: 10
+        # domain: <domain>  # default 'dynatrace'
+        # environment: <environment>  # default 'production'
+        # relative_time : <relative_time> # default 'hour'
+        # custom_device_relative_time: 1h
+        # custom_device_fields: +fromRelationships,+toRelationships,+tags,+managementZones,+properties.dnsNames,+properties.ipAddress
+        # tags:
+        #   - foo:bar
```
Comment (gggina) [inline stackpacks/integrations/dynatrace.md:68]: misses descriptions for some of the optional configuration:
- verify
- cert
- keyfile
- timeout
- relative_time
- custom_device_relative_time
- custom_device_fields
- tags

File: stackpacks/integrations/dynatrace.md [hunk 5]
```diff
@@ -35,36 +35,72 @@ Install the Dynatrace StackPack from the StackState UI **StackPacks** &gt; **Int
...
+    
+    ```
+2. Optional: Add a **domain** and **environment** in the `conf.yaml` file to specify where imported Dynatrace topology will end up in StackState (default domain=dynatrace and environment=production).
+3. [Restart the StackState Agent\(s\)](https://l.stackstate.com/ui-stackpack-restart-agent) to apply the configuration changes.
 4. Once the Agent has restarted, wait for data to be collected from Dynatrace and sent to StackState.
 
+#### Dynatrace Health Check
+
+To enable the Dynatrace health check and begin getting events from Dynatrace, add the following configuration to StackState Agent V2:
```
Comment (gggina) [inline stackpacks/integrations/dynatrace.md:77]: ..."and begin **collecting** events from Dynatrace"...

File: stackpacks/integrations/dynatrace.md [hunk 6]
```diff
@@ -35,36 +35,72 @@ Install the Dynatrace StackPack from the StackState UI **StackPacks** &gt; **Int
...
+        token: <token>  # API-Token to connect to Dynatrace
+        # verify: True  # By default its True
+        # cert: /path/to/cert.pem
+        # keyfile: /path/to/key.pem
+        # timeout: 10
+        # events_bootstrap_days: 5  # by default it's 5 days
+        # events_process_limit: 10000  # by default it's 10k events
+        # tags:
+        #   - foo:bar
+
```
Comment (gggina) [inline stackpacks/integrations/dynatrace.md:99]: misses descriptions for optional configuration:

- verify
- cert
- keyfile
- timeout
- event_bootstrap_days
- event_process_limit
- tags

**Review by gggina — COMMENTED** [review]
Also, the bullet point at line 13 should be split into 2 items:

- Agent V2 connects to the configured [Dynatrace API](dynatrace.md#rest-api-endpoints) to retrieve data:
   - If a Dynatrace topology check is configured, Smartscape topology data is retrieved.
   - If a Dynatrace health check is configured, events data is retrieved.

File: stackpacks/integrations/dynatrace.md (hunk 1 above)
Comment (hruhek) [reply stackpacks/integrations/dynatrace.md:38]: Done

File: stackpacks/integrations/dynatrace.md (hunk 2 above)
Comment (hruhek) [reply stackpacks/integrations/dynatrace.md:44]: Done

File: stackpacks/integrations/dynatrace.md (hunk 3 above)
Comment (hruhek) [reply stackpacks/integrations/dynatrace.md:71]: Done

File: stackpacks/integrations/dynatrace.md (hunk 4 above)
Comment (hruhek) [reply stackpacks/integrations/dynatrace.md:68]: done

File: stackpacks/integrations/dynatrace.md (hunk 5 above)
Comment (hruhek) [reply stackpacks/integrations/dynatrace.md:77]: Done

File: stackpacks/integrations/dynatrace.md (hunk 6 above)
Comment (hruhek) [reply stackpacks/integrations/dynatrace.md]: Done

**Pull request merged**